website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -8,142 +8,203 @@ pagination_prev: null
+
+Let's say you have a new employee being onboarded into your organization using [Okta](/docs/cloud/manage-access/set-up-sso-okta) as the IdP and dbt Cloud groups with SSO mappings. In this scenario, users are working on `The Big Project` and a new analyst named `Euclid Ean` is joining the group.
+
+Check out the following example configurations for an idea of how you can implement RBAC for your organization (these examples assume you have already configured [SSO](/docs/cloud/manage-access/sso-overview)):
+
+<Expandable alt_header="Okta configuration"> 
+
+You and your IdP team add `Euclid Ean` to your Okta environment and assign them to the `dbt Cloud` SSO app via a group called `The Big Project`. 
 
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/okta-group-config.png" width="60%" title="The user in the group in Okta." />
 
-### Permission sets
+Configure the group attribute statements the `dbt Cloud` application in Okta. The group statements in the following example are set to the group name exactly (`The Big Project`), but yours will likely be a much broader configuration. Companies often use the same prefix across all dbt groups in their IdP. For example `DBT_GROUP_`
 
-Permission sets are predefined collections of granular permissions. Permission
-sets combine low-level permission grants into high-level roles that can be
-assigned to groups. Some examples of existing permission sets are:
- - Account Admin
- - Git Admin
- - Job Admin
- - Job Viewer
- - ...and more
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-attributes.png" width="60%" title="Group attributes set in the dbt Cloud SAML 2.0 app in Okta." />
 
-For a full list of enterprise permission sets, see [Enterprise Permissions](/docs/cloud/manage-access/enterprise-permissions).
-These permission sets are available for assignment to groups and control the ability
-for users in these groups to take specific actions in the dbt Cloud application.
+</Expandable>
 
-In the following example, the _dbt Cloud Owners_ group is configured with the
-**Account Admin** permission set on _All Projects_ and the **Job Admin** permission
-set on the _Internal Analytics_ project.
+<Expandable alt_header="dbt Cloud configuration"> 
 
-<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-permissions.png" width="65%" title="Configuring permissions for the Admins group"/>
+You and your dbt Cloud admin team configure the groups in your account's settings: 
+1. Navigate to the **Account settings** and click **Groups & Licenses** on the left-side menu. 
+2. Click **Create group** or select and existing group and click **Edit**.
+3. Enter the group name in the **SSO** field.
+4. Configure the **Access and permissions** fields to your needs. Select a [permission set](/docs/cloud/manage-access/enterprise-permissions), the project-level access, and [environment-level access](/docs/cloud/manage-access/environment-permissions).
```

Comment:
Does project-level access mean they are just selecting the project the person will have access to? If so, maybe we can just say "Project they will access" or something similar?
```suggestion
4. Configure the **Access and permissions** fields to your needs. Select a [permission set](/docs/cloud/manage-access/enterprise-permissions), the project they can access, and [environment-level access](/docs/cloud/manage-access/environment-permissions).
```